src/lib/plugin-verify-github.js

```diff
@@ -0,0 +1,29 @@
+const parseSlug = require('parse-github-repo-url')
```

Comment:
I like how this code was pulled into its own module, thereby allowing GitLab developer to disable it on a per-project basis.